cmd2.py

```diff
@@ -276,6 +276,24 @@ def cmd_wrapper(instance, cmdline):
     return arg_decorator
 
 
+def with_argument_list(func):
```

Comment:
I like this decorator-based approach for getting ``cmd2`` to pass the do_* methods an argument list instead of an argument string.  It is an easy way to alter the default behavior on a per-function basis.